src/PowerShellEditorServices/Debugging/DebugService.cs

```diff
@@ -58,6 +58,20 @@ public DebugService(PowerShellContext powerShellContext)
         #region Public Methods
 
         /// <summary>
+        /// Disables all breakpoints in the runspace.
+        /// </summary>
+        /// <returns></returns>
+        public async Task DisableAllBreakpoints()
```

Comment:
Argh...  Does it look like the script is getting executed on our end?  Are they just not showing our output in the UI?